index.js

```diff
@@ -0,0 +1,91 @@
+function gameResult() {
+  //week 1
+  var eaglesWK1W = document.querySelector("#wk1-w");
+  var eaglesWK1L = document.querySelector("#wk1-l");
+  // var eaglesWK1T = document.querySelector('.tie');
+  var eaglesWK1Score = document.querySelector("#wk1-home");
+  var nyg1WK1Score = document.querySelector("#wk1-away");
+  var tieWK1Score = document.querySelector("#wk1-t");
+  if (eaglesWK1Score.innerHTML > nyg1WK1Score.innerHTML) {
+    eaglesWK1W.innerHTML = 1;
+    eaglesWK1L.innerHTML = 0;
+  } else if (eaglesWK1Score.innerHTML < nyg1WK1Score.innerHTML) {
+    eaglesWK1W.innerHTML = 0;
+    eaglesWK1L.innerHTML = 1;
+  } else if (eaglesWK1Score.innerHTML === nyg1WK1Score.innerHTML) {
+    eaglesWK1T.style.display = "flex";
+    tieWK1Score.innerHTML = 1;
+  }
+
+  //week 2
```

Comment:
So there is a lot of repetitive code being used throughout the app.

Create a single function that handles all of the JS DOM manipulation, and takes in the appropriate selectors.

So for each week:

```
// Week 1
generateScore({
  winStatus: '#wk1-w",
  lossStatus:  '#wk1-l',
  tieStatus: null, // if not available
  homeScore: '#...',
  awayScore: '#...',
  tieScore: '#...'
})

// Week 2
generateScore({
  winStatus: '#wk2-w",
  lossStatus:  '#wk2-l',
  tieStatus: null, // if not available
  homeScore: '#...',
  awayScore: '#...',
  tieScore: '#...'
})


... etc
```

There is also a way to trim down the amount of selectors, but for now, try to accomplish this.